osu.Game/Overlays/Comments/GetCommentRepliesButton.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Framework.Allocation;
+using osu.Game.Graphics.UserInterface;
+using System.Collections.Generic;
+using osuTK;
+using osu.Game.Online.API;
+using osu.Framework.Bindables;
+using osu.Game.Online.API.Requests;
+using System.Linq;
+
+namespace osu.Game.Overlays.Comments
+{
+    public abstract class GetCommentRepliesButton : LoadingButton
+    {
+        public readonly Bindable<List<Comment>> ChildComments = new Bindable<List<Comment>>();
```

Comment:
I'm not really a fan of this, modifying `ChildComments` from a button. I'd much rather have this button expose an event like `Action<List<Comment>> OnCommentsReceived`